<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HIP_Gui_MainWindow</class>
 <widget class="QMainWindow" name="HIP_Gui_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="_main_window_w">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="_splitter_w">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="_left_layout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="_selector_w" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="_explorer_frame_w" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="_explorer_splitter_w">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="handleWidth">
              <number>10</number>
             </property>
             <widget class="QWidget" name="_explorer_w" native="true"/>
             <widget class="QWidget" name="_point_editor_w" native="true"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="_tab_w">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="_menu_bar_w">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="_menu_file_w">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_action_export_database"/>
    <addaction name="_action_exit"/>
   </widget>
   <widget class="QMenu" name="_menu_help_w">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="_action_about"/>
   </widget>
   <addaction name="_menu_file_w"/>
   <addaction name="_menu_help_w"/>
  </widget>
  <widget class="QStatusBar" name="_status_bar_w"/>
  <widget class="QToolBar" name="_toolbar_w">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="_action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="_action_about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="_action_export_database">
   <property name="text">
    <string>Export database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
